src/components/cylc/table/Table.vue

```diff
@@ -0,0 +1,247 @@
+              id="c-table-filter-task-states"
+              :items="taskStates"
+              clearable
+              dense
+              flat
+              hide-details
+              multiple
+              outlined
+              placeholder="Filter by task state"
+              v-model="tasksFilter.states"
+            >
+              <template v-slot:item="slotProps">
+                <Task :status="slotProps.item.value"/>
+                <span class="ml-2">{{ slotProps.item.value }}</span>
+              </template>
+              <template v-slot:selection="slotProps">
+                <div class="mr-2" v-if="slotProps.index >= 0 && slotProps.index < maximumTasks">
+                  <Task :status="slotProps.item.value"/>
+                </div>
+                <span
+                  v-if="slotProps.index === maximumTasks"
+                  class="grey--text caption"
+                >
+            (+{{ tasksFilter.states.length - maximumTasks }})
+          </span>
+              </template>
+            </v-select>
+          </v-col>
+          <v-col
+            cols="12"
+            md="2">
+            <!-- TODO: we shouldn't need to set the height (px) here, but for some reason the Vuetify
+                       components don't seem to agree on the height here -->
+            <v-btn
+              id="c-table-filter-btn"
+              height="40"
+              block
+              outlined
+              @click="filterTasks"
+            >Filter</v-btn>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+    <table layout="auto" width="100%" padding="1px" border="1px solid black">
+      <thead>
+      <tr align="left">
+        <th>Task</th>
+        <th>State</th>
+        <th>Cyclepoint</th>
+        <th>Host</th>
+        <th>Job System</th>
+        <th>Job ID</th>
+        <th>T-submit</th>
+        <th>T-start</th>
+        <th>T-finish</th>
+        <th>dT-mean</th>
+      </tr>
+      </thead>
+      <tr
+          v-for="task of this.tasks"
+          :key="task.id"
+          >
+        <td>{{ task.id }}</td>
+        <td>{{ task.state }}</td>
+        <td>{{ task.cyclePoint }}</td>
+        <td>{{ getTaskJobProps(task, 'platform') }}</td>
+        <td>{{ getTaskJobProps(task, 'jobRunnerName') }}</td>
+        <td>{{ getTaskJobProps(task, 'jobId') }}</td>
+        <td>{{ getTaskJobProps(task, 'submittedTime') }}</td>
+        <td>{{ getTaskJobProps(task, 'startedTime') }}</td>
+        <td>{{ getTaskJobProps(task, 'finishedTime') }}</td>
+        <td>{{ task.meanElapsedTime }}</td>
+      </tr>
+    </table>
+  </v-container>
+</template>
+
+<script>
+import TaskState from '@/model/TaskState.model'
+import Task from '@/components/cylc/Task'
+import clonedeep from 'lodash.clonedeep'
+
+export default {
+  name: 'TableComponent',
+  props: {
+    tasks: {
+      type: Array,
+      required: true
+    },
+    hoverable: Boolean,
+    activable: Boolean,
+    multipleActive: Boolean,
```

Comment:
We can remove `hoverable`, `activable`, and `multipleActive` props here, and in the caller code (i.e. `views/Table.vue`).